src/styles/main.scss

```
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import "../styles/block/reset.scss";
```

fix it for all imports
```suggestion
@import "./block/reset.scss";
```